### DISCUSSION TITLE
Check if running app version is the `published` version or `dev` version

### DISCUSSION THREAD

**Discussion by SantoshSrinivas79** [Budibase/budibase#6442] [Help]
I want to set some config params that will change based on whether it is the `dev` version or `live` version.

I have a `site` table with the different parameters

Anyway I can get the version in the UI?

Also, is there a variable I can use for the current app `ROOT URL`? (both in dev and prod)

**Comment by melohagan** [comment]
Hi @SantoshSrinivas79 

Unfortunately I don't think bindings for these options are currently available. 

Feel free to request a feature with a description of what you hope to achieve! https://github.com/Budibase/budibase/issues/new/choose
